src/scripts/rechunk.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+import dask.array as da
+import argparse
+import time
+
+
+def timeit(method):
+    def timed(*args, **kw):
+        ts = time.time()
+        result = method(*args, **kw)
+        te = time.time()
+        if 'log_time' in kw:
+            name = kw.get('log_name', method.__name__.upper())
+            kw['log_time'][name] = int((te - ts) * 1000)
+        else:
+            print('%r  %2.2f ms' %
```

Comment:
```suggestion
            print('%s  %2.2f ms' %
```

Reply:
`%r` is for using `repr()`. I'm not exactly sure why the snippet author chose that, but it certainly doesn't hurt.

Reply:
Basically I wondered why we want to see quotes around the printed words.